numba/npyufunc/tbbpool.cpp

```diff
@@ -0,0 +1,105 @@
+/*
+Implement parallel vectorize workqueue on top of Intel TBB.
+*/
+
+#define TBB_PREVIEW_WAITING_FOR_WORKERS 1
+#include <tbb/tbb.h>
+#include <string.h>
+#include <stdio.h>
+#include "workqueue.h"
+#include "../_pymodule.h"
+
+static tbb::task_group *tg;
+static tbb::task_scheduler_init *tsi;
```

Comment:
IIRC, C/C++ standard says static pointer variables is initialized to null, right?